db/schema.rb

```diff
@@ -166,18 +158,19 @@
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "user_id", null: false
     t.bigint "patient_id", null: false
-    t.bigint "sick_leave_id", null: false
+    t.bigint "sick_leave_id"
+    t.datetime "next_visit_at"
```

Comment:
it seems not scalable. Probably it's worth linking on the whole entity like `visit.next_visit`, cause usually patients are aware not only about the time of the next appointment but a place/a doctor, etc

what do you think?